RELEASE_NOTES.md

```diff
@@ -10,16 +10,17 @@ This serves two purposes:
 2. At release time, you can move the Unreleased section changes into a new release version section.
 
 ### Added
-- for new features.
+- Add experimental `InMemoryPage::getOutputPath` overload support
 
 ### Changed
-- for changes in existing functionality.
+- Changed how the documentation search is generated, to be an `InMemoryPage` instead of a post-build task.
+- Updated the documentation article component to support existing `$document` instance.
```

Comment:
May want to cherry pick this after further testing. We may also want to consider renaming the variable, but need to think about if that is a breaking change.

Reply:
Since it's just a local variable inside a single Blade file I can't imagine it's very breaking, unless someone includes a child template within an extended version of our layout but (famous last words) I don't see why anyone would want to do that (and actually if they extend the template they would not get the change)

Reply:
https://github.com/hydephp/develop/pull/1506